tests/test_cvpr_conference_v2.py

```diff
@@ -926,3 +926,22 @@ def test_metareview_revision_stage(self, client, openreview_client, helpers, tes
         assert 'award_candidate' in meta_review.content
 
         assert meta_review.content['metareview']['value'] == 'Revised comment title'
+
+        # Secondary AC can't post meta review revision
+        secondary_ac_client = openreview.api.OpenReviewClient(username='ac1@cvpr.cc', password=helpers.strong_password)
+        secondary_ac_anon_group_id = secondary_ac_client.get_groups(prefix=f'thecvf.com/CVPR/2024/Conference/Submission4/Secondary_Area_Chair_.*', signatory='ac1@cvpr.cc')[0].id
+
+        with pytest.raises(openreview.OpenReviewException, match=r'signatures can only contain the allowed values: thecvf.com/CVPR/2024/Conference/Program_Chairs, thecvf.com/CVPR/2024/Conference/Submission4/Area_Chair_.*'):
```

Comment:
I was expecting to see an error like "user is not an invitee..." may be we need to merge Celeste's PR first